### PR TITLE
Copy over the osbuilder-tools image assets and merge with auroraboot

### PR DESCRIPTION
**Pull request by jimmykarily**
So we can deprecate the [osbuilder-tools image](https://github.com/kairos-io/osbuilder/tree/master/tools-image) in favor of this one.

Part of https://github.com/kairos-io/kairos/issues/1633

### CODE REVIEW

File: Earthfile [hunk 1]
```diff
@@ -71,3 +82,27 @@ test-bootable:
     ARG CREATE_VM=true
     RUN date
     RUN go run github.com/onsi/ginkgo/v2/ginkgo run --label-filter "bootable" -v --fail-fast -r ./e2e
+
+last-commit-packages:
+    FROM quay.io/skopeo/stable
+    RUN dnf install -y jq
+    WORKDIR build
+    ENV jqQuery='.Tags | map(select(. | contains("-repository.yaml"))) | sort_by(. | sub("v";"") | sub("-repository.yaml";"") | sub("-git.*";"") | .[0:12] | tonumber) | .[-1]'
```
Comment (jimmykarily) [inline Earthfile:90]: This was broken in enki from the moment when our images started having `git` in their names. I fixed it here.

File: renovate.json [hunk 2]
```diff
@@ -18,5 +20,16 @@
...
     }
+  ],
+  "regexManagers": [
+    {
+      "fileMatch": [
+        "image-assets/luet-amd64.yaml$",
+        "image-assets/luet-arm64.yaml$"
+      ],
+      "matchStrings": [
+        "#\\s*renovate:\\s*datasource=(?<datasource>.*?) depName=(?<depName>.*?)?\\s+reference:\\s(?<currentValue>.*?)\\s"
```
Comment (jimmykarily) [inline renovate.json:31]: I don't know if this works. I just copied it from enki. We'll see.

File: Dockerfile [hunk 3]
```diff
@@ -1,19 +1,24 @@
...
 ADD go.mod .
 ADD go.sum .
 RUN go mod download
 ADD . .
-RUN CGO_ENABLED=0 go build -ldflags "-X main.version=${BINARY_VERSION}" -o auroraboot
-
-FROM quay.io/kairos/osbuilder-tools:$VERSION
+ENV CGO_ENABLED=0
+ENV VERSION=$VERSION
+RUN go build -buildvcs=false -ldflags "-X main.version=${VERSION}" -o auroraboot
```
[the remark this answers is not in the document]
Comment (Itxaka) [reply Dockerfile:15]: Wat, lol.

**Review by Itxaka — COMMENTED** [review]
Looks good, let's see what we break 😂

**Review by Itxaka — APPROVED** [review]

File: Dockerfile [hunk 4]
```diff
@@ -1,19 +1,24 @@
...
 
+FROM opensuse/leap:$LEAP_VERSION AS default
+RUN zypper ref && zypper dup -y
+## ISO+ Arm image + Netboot + cloud images Build depedencies
+RUN zypper ref && zypper in -y bc qemu qemu-tools jq cdrtools docker git curl gptfdisk kpartx sudo xfsprogs parted binutils \
```
Comment (Itxaka) [inline Dockerfile:20]: can we add a TODO here to track grub artifacts here and below? We may not need some of them anymore with aurora bundling the iso cd images

File: Dockerfile (hunk 3 above)
[the remark this answers is not in the document]
Comment (jimmykarily) [reply Dockerfile]: removed with a new commit (with explanation)

**Pull request merged**